db/db_memtable_test.cc

```diff
@@ -339,6 +339,135 @@ TEST_F(DBMemTableTest, ColumnFamilyId) {
   }
 }
 
+class DBMemTableTestForSeek : public DBMemTableTest,
+                              virtual public ::testing::WithParamInterface<
+                                  std::tuple<bool, bool, bool>> {};
+
+TEST_P(DBMemTableTestForSeek, IntegrityChecks) {
+  // Validate key corruption could be detected during seek.
+  // We insert many keys into skiplist. Then we sequentially corrupt each one of
```

Comment:
I read "sequentially corrupt each one of them" to mean all keys would be corrupted at the same time. Perhaps "corrupt the each key one at a time."